README.md

```diff
@@ -16,10 +19,20 @@ Deploying in another environment may require additional configuration; previous
 You may use the Docker image for testing during development as described above, but it is not required.
```

Comment:
```suggestion
```
This line should be removed. There is sentence below with a link to the wiki for the Docker image dev.